modules/using-assisted-installer-oci-agent-iso.adoc

```diff
@@ -6,62 +6,80 @@
 [id="using-assisted-installer-oci-agent-iso_{context}"]
 = Using the {ai-full} to generate an {oci}-compatible discovery ISO image
 
-Generate a discovery ISO image and upload the image to {oci-first}, so that the agent can perform hardware and network validation checks before you install an {product-title} cluster on {oci}.
+Create the cluster configuration and generate the discovery ISO image in the {ai-full} web console. 
 
-From the {oci} web console, you must create the following resources:
+.Prerequisites
 
-* A compartment for better organizing, restricting access, and setting usage limits to {oci} resources.
-* An object storage bucket for safely and securely storing the discovery ISO image. You can access the image at a later stage for the purposes of booting the instances, so that you can then create your cluster.
+* You created a child compartment and an object storage bucket on {oci}. For details, see _Preparing the {oci} environment_. 
+* You reviewed details about the {product-title} installation and update processes. 
 
-.Prerequisites
+== Creating the cluster
```

Comment:
This change splits the procedure into two procedures. This requires two procedure modules (can't have two sets of `.Procedure` steps)